notebooks/book2/17/gpKernelPlot.ipynb

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.      periodic_kernel = kernels.ExpSineSquared(scale = 1.0,gamma=jnp.array(2.0))
</pre><p>This looks too streched. Maybe increase the period so that it looks good comparable to matern plots.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
![image](https://user-images.githubusercontent.com/68188848/176156758-0a66efd2-e679-494f-a589-57e4eef77018.png)
Is this any better?